QUARKUS-6243.md

```diff
@@ -0,0 +1,48 @@
+# QUARKUS-6243 - Dedicated Dev UI interface to execute HQL (Hibernate ORM) queries
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6243
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/46728
+- https://github.com/quarkusio/quarkus/pull/49165 
+
+PR #46728 introduces a new HQL Console in the Quarkus Dev UI for Hibernate ORM.
+It allows developers to select Hibernate persistence units and their entities, displaying existing data and executing custom HQL queries.
+
+## Scope of the testing
+QE team will:
+- Get familiar with the HQL Console functionality using video preview from PR #46728
+- Review upstream test coverage provided as part of #46728
+- Explore possible enhancements of the existing upstream test coverage
+- Confirm that no additional coverage is needed in QE Test Suite since this feature is dev-support only
+
+## Getting familiar with the feature
+- Understand how the HQL Console integrates in Dev UI: 
+  - Query execution 
+  - Display of results
+  - Display of error messages
+
+## Test coverage
+Since the feature is dev-support scoped, coverage can be extended only in upstream `devmode` integration tests.
+
+Additional scenarios to be included:
+- Valid query execution
+  - Insert test entities and run a valid HQL query
+  - Assert JSON response contains expected rows
+
+- Invalid query
+  - Submit an invalid HQL query and assert returned error message
+
+- Nested/complex values
+  - Entities with associations or JSON fields
+  - Assert nested results are serialized as expected
```

Comment:
what this sentence means practically? I need something to imagine under it, thank you (no need to update TP yet, first I need to understand)